src/data/faq.json

```diff
@@ -305,7 +305,7 @@
                         "textblock": [
                             "Together with the European Commission, an infrastructure for secure information exchange has been developed, known as the \"Interoperability Gateway\". This gateway enables the Corona Warn App to operate across national borders and to receive potential warning messages abroad. Detailed information about the Gateway is available from the European Commission in <a href='https://ec.europa.eu/commission/presscorner/detail/en/qanda_20_1905#gateway'>Coronavirus: EU interoperability gateway for contact tracing and warning apps – Questions and Answers</a>.",
                             "To view more information in the app, tap",
-			    "<ol><li>'EXPOSURE LOGGING ACTIVE/STOPPED'</li><li>Then tap 'Transnational Exposure Logging'.</li><li>At last click on 'Participating Countries'.</li></ol>",
+			                "<ol><li>'EXPOSURE LOGGING ACTIVE/STOPPED'</li><li>Then tap 'Transnational Exposure Logging'.</li><li>At last scroll down to the list of countries.</li></ol>",
```

Comment:
Please remove "At last". It is not needed in a bullet list and it is incorrect English usage because it means after a long wait.
Then the third bullet point becomes:

"Scroll down to the list of countries."

"At last" means "after a long wait".
See https://dictionary.cambridge.org/grammar/british-grammar/finally-at-last-lastly-or-in-the-end.